packages/lib/server-only/document/complete-document-with-token.ts

```diff
@@ -49,8 +49,8 @@ export const completeDocumentWithToken = async ({
 
   const document = await getDocument({ token, documentId });
 
-  if (document.status === DocumentStatus.COMPLETED) {
```

Comment:
Isn't this check needed anywhere anymore?

Reply:
What do you mean? The check is still there indirectly via `status !== PENDING`

Reply:
Got confused with the statuses. You right.